easybuild/easyconfigs/x/xcb-proto/xcb-proto-1.11.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'xcb-proto'
+version = '1.11'
+
+homepage = 'http://xcb.freedesktop.org/'
+description = """The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint,
+latency hiding, direct access to the protocol, improved threading support, and extensibility."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[0:2])
+
+sanity_check_paths = {
+    'files': ['lib/pkgconfig/xcb-proto.pc'],
+    'dirs': ['lib/python%s/site-packages/xcbgen' % pyshortver]
```

Comment:
@boegel: this handled by the `pkgconfig`:
`xcb-proto.pc`:
```
prefix=/apps/brussel/ivybridge/software/xcb-proto/1.11
exec_prefix=${prefix}
datarootdir=${prefix}/share
datadir=${datarootdir}
libdir=${exec_prefix}/lib
xcbincludedir=${datadir}/xcb
pythondir=${prefix}/lib/python2.6/site-packages

Name: XCB Proto
Description: X protocol descriptions for XCB
Version: 1.11
```

I am more concerned what will happen with the precompiled pyo files if you change `Python` version. (especially, if they are read-only)